client-cli/src/command/address_command.rs

```diff
@@ -123,11 +129,26 @@ impl AddressCommand {
                 success(&format!("New address: {}", address));
                 Ok(())
             }
+            AddressType::StakingWatch => {
+                let enckey = ask_seckey(None)?;
+                let public_key = ask_public_key(None)?;
```

Comment:
public key is not secret stuff, maybe pass it in the argument?

Reply:
sure

Reply:
But when creating a normal staking address or transfer address, no need to pass the pubkey, if we pass the pubkey in the arguments, it may make some confusion.

Reply:
It's more like import an watch-only address, rather than create an address?

Reply:
The staking address and its state is public, anyone with the staking address is able to check its state.